From josef.jiru@esk.fraunhofer.de  Wed Mar 11 13:54:57 2009
From: "Josef Jiru" <josef.jiru@esk.fraunhofer.de>
Date: Wed, 11 Mar 2009 15:50:48 +0100
Subject: Staging: rt2870: add Linksys WUSB600N device id
To: "Greg KH" <gregkh@suse.de>
Message-ID: <EAFC716668964A48A4B97DD4703A4E604D742D@mara.esk.fraunhofer.de>


Updated usb device list to support Linksys WUSB600N wireless adapter

Signed-off-by: Josef Jiru <josef.jiru@gmx.net>


---
 drivers/staging/rt2870/rt2870.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rt2870/rt2870.h
+++ b/drivers/staging/rt2870/rt2870.h
@@ -86,6 +86,7 @@
 #define RT2870_USB_DEVICES	\
 {	\
 	{USB_DEVICE(0x148F,0x2770)}, /* Ralink */		\
+	{USB_DEVICE(0x1737,0x0071)}, /* Linksys WUSB600N */	\
 	{USB_DEVICE(0x148F,0x2870)}, /* Ralink */		\
 	{USB_DEVICE(0x148F,0x3070)}, /* Ralink */		\
 	{USB_DEVICE(0x0B05,0x1731)}, /* Asus */			\
